src/server/game/DataStores/DB2Stores.cpp

```diff
@@ -917,6 +925,12 @@ void DB2Manager::LoadStores(std::string const& dataPath, uint32 defaultLocale)
     for (WorldMapAreaEntry const* worldMapArea : sWorldMapAreaStore)
         _worldMapAreaByAreaID[worldMapArea->AreaID] = worldMapArea;
 
+    for (RewardPackXItemEntry const* rewardPackXItem : sRewardPackXItemStore)
+        _rewardPackXItems[rewardPackXItem->RewardPackID].push_back(rewardPackXItem);
+
+    for (PvpRewardEntry const* pvpReward : sPvpRewardStore)
+        _pvpRewardPack[std::pair<uint32, uint32>{ pvpReward->Prestige, pvpReward->HonorLevel }] = pvpReward->RewardPackID;
```

Comment:
On local it does with VS2017 and it looks like travis is complaining about something else :p